packages/nuxt/src/app/plugins/payload.client.ts

```diff
@@ -26,5 +18,17 @@ export default defineNuxtPlugin({
       if (!payload) { return }
       Object.assign(nuxtApp.static.data, payload.data)
     })
+
+    onNuxtReady(() => {
+      // Load payload into cache
+      nuxtApp.hooks.hook('link:prefetch', async (url) => {
+        if (!parseURL(url).protocol) {
+          await loadPayload(url)
+        }
+      })
+      if (navigator.connection?.effectiveType !== 'slow-2g') {
+        setTimeout(getAppManifest, 1000)
```

Comment:
Is this 1s arbitrary? 

If so, there is a specific threshold you need to exceed for the script not to be considered blocking (different for devices), if this isn't passing that, maybe it's worth just loading immediately on Nuxt ready?